### PR TITLE
Added team information and docker image for UBC Thunderbots

### PR DESCRIPTION
**Pull request by PhilD71**
Updated `docker-compose-teams,yml` and `config/teams` to add UBC Thunderbots information

### CODE REVIEW

File: docker-compose-teams.yaml [hunk 1]
```diff
@@ -36,9 +36,27 @@ services:
       - match
       - guacamole
 
+  team-ubcthunderbots:
+    image: ubcthunderbots/tbots-software-env:0.0.1
```
Comment (g3force) [inline docker-compose-teams.yaml:40]: Do you have the image publicly available, so that I can have a quick look at it?

Comment (PhilD71) [reply docker-compose-teams.yaml:40]: Image is hosted here: https://hub.docker.com/r/ubcthunderbots/tbots-software-env

Let me know if you need the dockerfile itself (I think you should be able to view it on dockerhub)

Comment (g3force) [reply docker-compose-teams.yaml:40]: Yeah, I wanted to have a look at the Dockerfile, but the source is not shown on Docker Hub, only the raw layers.
You probably just pushed the image yourself, without connecting a GitHub Repo with the Docker Hub Repo.

Comment (g3force) [reply docker-compose-teams.yaml:40]: oh, I've found it through the connected issue above :D 

Looks fine :+1:

**Review by g3force — APPROVED** [review]

**Pull request merged**